.env

```diff
@@ -0,0 +1 @@
+PHP_VERSION='8.0'
```

Comment:
This should be done on the command line, not in an `.env` file.

Don't really need to introduce `.env` files just for this.

Reply:
nvm, just seeing https://docs.docker.com/compose/environment-variables/

Reply:
Fixed https://github.com/google/web-stories-wp/pull/6090/commits/f5f5c9e13b94fee6d397b1146c929fda04c85f49